src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.FrameworkReferenceResolution.targets

```diff
@@ -303,7 +303,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     ============================================================
     -->
   <Target Name="ResolveRuntimePackAssets" DependsOnTargets="ResolveFrameworkReferences"
-          Condition="'@(RuntimePack)' != '' And '$(DesignTimeBuild)' != 'true'">
+          Condition="'@(RuntimePack)' != '' And ('$(DesignTimeBuild)' != 'true' or '$(ResolveRuntimeAssetsDuringDesignTimeBuild)' == 'true')">
```

Comment:
Rather than adding the `ResolveRuntimeAssetsDuringDesignTimeBuild` property and the `BeforePublishItemsOutputGroup` target, I think we can just delete the `'$(DesignTimeBuild)' != 'true'` clause on this condition.

Reply:
Fixed